letta/server/server.py

```diff
@@ -1363,6 +1373,38 @@ def list_embedding_models(self) -> List[EmbeddingConfig]:
                 warnings.warn(f"An error occurred while listing embedding models for provider {provider}: {e}")
         return embedding_models
 
+    def get_llm_config_from_handle(self, handle: str, context_window: Optional[int] = None) -> LLMConfig:
+        provider_name, model_name = handle.split("/", 1)
+        model_name, _, context_window_override = model_name.partition(":")
+        providers = [provider for provider in self._enabled_providers if provider.name == provider_name]
+        if not providers:
+            raise ValueError(f"Provider {provider_name} is not supported")
+
+        llm_configs = [llm_config for llm_config in providers[0].list_llm_models() if llm_config.model == model_name]
+        if not llm_configs:
+            raise ValueError(f"LLM model {model_name} is not supported by {provider_name}")
+
+        llm_config = llm_configs[0]
```

Comment:
maybe we should throw some kind of error or warning here if `len(llm_configs) > 1`? and similar for embedding config

Reply:
Split will throw the error for this case (based on the limit param I specified), do you prefer a custom error for this case? Probably pretty unlikely that we have multiple matches since we maintain this internally right